Sources/RoutingKit/Routing/TrieRouter+Node.swift

```diff
@@ -87,6 +91,16 @@ extension TrieRouter {
     }
 }
 
+extension TrieRouter.Node: Hashable {
```

Comment:
What do we need this conformance for?

Reply:
We don't originally I was going to make constants a set of nodes instead, but I decided against it. Removed